index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Se eu tivesse um sobrenome desses eu colocava ele em todo lugar... kkkkk

```suggestion
    <title>Beatriz Estebanez - Lessons Learned</title>
```

Reply:
Obrigada Wanderson! Já commitei sua sugestão (: